qr-code/src/main/java/com/adyen/checkout/qrcode/internal/ui/DefaultQRCodeDelegate.kt

```diff
@@ -152,18 +153,14 @@ internal class DefaultQRCodeDelegate(
             return
         }
 
-        val viewType = when (action.paymentMethodType) {
-            PaymentMethodTypes.PAY_NOW -> {
-                maxPollingDurationMillis = PAY_NOW_MAX_POLLING_DURATION
-                QrCodeComponentViewType.FULL_QR_CODE
-            }
-            PaymentMethodTypes.UPI_QR -> {
-                maxPollingDurationMillis = UPI_MAX_POLLING_DURATION
-                QrCodeComponentViewType.FULL_QR_CODE
-            }
-            else -> {
-                maxPollingDurationMillis = DEFAULT_MAX_POLLING_DURATION
-                QrCodeComponentViewType.SIMPLE_QR_CODE
+        var viewType = QrCodeComponentViewType.SIMPLE_QR_CODE
```

Comment:
It's better to have a `DEFAULT` in the enum with these values to keep everything in the same place. We can return `DEFAULT` if we can't find any matching enum entry.

Reply:
I thought it but I didn't do it because we might face the same problem like UNKNOWN in CardType and we don't have a payment method type for the default value.

Reply:
Yeah but the use case is a bit different here. For instance in `AddressSpecification` we have `DEFAULT` to represent a ui state. I believe it's better to have `DEFAULT` as an entry.